CleffyNotes/ViewControllers/MetronomeViewController.swift

```diff
@@ -0,0 +1,236 @@
+    
+    var tempo: Int = 80 {
+        didSet {
+            metronome.tempo = Double(tempo)
+            metronome2.tempo = Double(tempo * 3)
+            timeSignatureController?.lastMetronomeSettings(tempo, index)
+        }
+    }
+    
+    var index: Int = 0 {
+        didSet {
+            timeSignatureController?.lastMetronomeSettings(tempo, index)
+            metronome.subdivision = timeSignatureController?.timeSignatures[index].beats ?? 2
+        }
+    }
+    
+    lazy var timeSignature: TimeSignature = timeSignatureController?.timeSignatures[index] ?? TimeSignature(name: "two4", image: UIImage(named: "two4"), beats: 2, displayBeats: 2)
+        
+    var beatArray: [Int] = [1,2]
+    var displayBeatIndex: Int = 0
+
+    @IBOutlet weak var beatsPerMinuteLabel: UILabel!
+    @IBOutlet weak var timeSignatureLabel: UILabel!
+    @IBOutlet weak var startButton: UIButton!
+    @IBOutlet weak var displayBeatLabel: UILabel!
+    
+    @IBOutlet weak var beatsSlider: UISlider!
+    @IBOutlet weak var timeSignaturePicker: UIPickerView!
+    
+    lazy var metronome: AKMetronome = {
+        let metronome = AKMetronome()
+        metronome.subdivision = timeSignature.beats
+        metronome.tempo = Double(tempo)
+        metronome.frequency1 = 450
+        metronome.frequency2 = 600
+        metronome.callback = {
+            DispatchQueue.main.async {
+                switch self.timeSignature.name {
+                case "three8", "six8", "nine8", "twelve8":
+                    var beatArray2: [Int] = []
+                    for beat in 1...self.timeSignature.beats {
+                        for _ in 1...4 {
+                            beatArray2.append(beat)
+                        }
+                    }
+                    self.displayBeatLabel.text = "\(beatArray2[self.displayBeatIndex])  :  \(self.beatArray[self.displayBeatIndex])"
+                    if self.displayBeatIndex < (self.beatArray.count - 1) {
+                        self.displayBeatIndex += 1
+                    } else {
+                        self.displayBeatIndex = 0
+                    }
+                default:
+                    self.displayBeatLabel.text = "\(self.beatArray[self.displayBeatIndex])"
+                    if self.displayBeatIndex < (self.beatArray.count - 1) {
+                        self.displayBeatIndex += 1
+                    } else {
+                        self.displayBeatIndex = 0
+                    }
+                }
+            }
+        }
+        return metronome
+    }()
+    
+    lazy var metronome2: AKMetronome = {
+        let metronome2 = AKMetronome()
+        metronome2.tempo = Double(tempo * 3)
+        metronome2.subdivision = timeSignature.beats
+        metronome2.frequency1 = 900
+        return metronome2
+    }()
+    
+    override func viewDidLoad() {
+        do {
+            try AudioKit.stop()
+        } catch {
+            print("error")
```

Comment:
You should prefer to print the actual error through string interpolation `\(error)` or at least give yourself a bit more of a description of the error like "Error stopping AudioKit", or a combination of those two.